src/test/fixtures/fake-feature-toggle-store.ts

```diff
@@ -128,16 +123,16 @@ export default class FakeFeatureToggleStore implements IFeatureToggleStore {
         });
     }
 
-    async getVariants(featureName: string): Promise<IVariant[]> {
+    async getVariants(featureName: string): Promise<VariantSchema[]> {
         const feature = await this.get(featureName);
-        return feature.variants;
+        return feature.variants as VariantSchema[];
```

Comment:
Are we sure this holds? If `IVariant` and `VariantSchema` don't match, it might be something we should fix.